src/dashboard/home-dashboard.component.tsx

```diff
@@ -10,9 +10,9 @@ export default function HomeDashboard(props: HomeDashboardProps) {
         <UserHasAccessReact privilege="View Patients">
           <Link
             to={props.match.url + "/patient-search"}
-            className="omrs-link omrs-outlined-action omrs-rounded"
+            className={styles.link}
```

Comment:
I would rather not write our own custom css when we have styleguide classes that already do this. Can you explain what these changes accomplish? Can any of the changes be moved to esm-styleguide?

Reply:
We were trying to expand the search button and center its content and yes most of the changes can be moved to the styleguide. I'll make the required changes right away.

Reply:
👍

Reply:
New fix up for review now.